### PR TITLE
add note about what sender id of 0x42 represents

### PR DESCRIPTION
**Pull request by denniszollo**
@mfine  What do you think about adding this to the spec to note what 0x42 represents?

### CODE REVIEW

**Comment by mfine** [comment]
Maybe something more like ```By default, clients of `libsbp` use a sender id value of `0x42````

File: spec/yaml/swiftnav/sbp/settings.yaml [hunk 1]
```diff
@@ -39,7 +39,9 @@ definitions:
           type: string
           desc: |
             A NULL-terminated and delimited string with contents
-            [SECTION_SETTING, SETTING, VALUE].
+            [SECTION_SETTING, SETTING, VALUE]. A device will only
+            react to this message when it is received from
```
Comment (mfine) [inline spec/yaml/swiftnav/sbp/settings.yaml]: /react to/process/

**Comment by mfine** [comment]
Couple of nits. Also do the file_io messages?

**Comment by mookerji** [comment]
@denniszollo can you rebase this so we can merge?

**Comment by denniszollo** [comment]
@mookerji - should be rebased now.  There used to be notes about how you know when you get to the "end" of a list.  Since those were removed previously, I left them out, but would actually prefer if we kept them in as it is useful information.

**Pull request merged**